.ci/cico_rhche_prcheck.sh

```diff
@@ -27,7 +27,11 @@ export PR_CHECK_BUILD=${PR_CHECK_BUILD:-"true"}
 function BuildTagAndPushDocker() {
   echo "Docker status:"
   docker images
-  .ci/cico_build.sh
+#  .ci/cico_build.sh
+  ABSOLUTE_PATH=$(cd $(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../ && pwd)
+  docker run --privileged=true -u root -v ${ABSOLUTE_PATH}/:/data/rhche/ quay.io/openshiftio/rhchestage-rh-che-automation-dep:latest scl enable rh-maven33 rh-nodejs4 "mvn -f /data/rhche/ clean install"
+  source ./config
```

Comment:
`./config` was already sourced on line 109. Are you sure you need to do that again?